README.md

```diff
@@ -154,6 +154,69 @@ IResult<string> result.Map(value => value.toString());
 
 If `result` represents an failure all calls to `Map` and `FlatMap` are ignored. See `FaunaDB.Types.Result`.
 
+### How to work with user defined classes
+
+Instead of create manually your objects using our dsl (via `Obj()` method) it's possible to use our `Encoder` class and convert any user defined class to an equivalent `Value` type.
```

Comment:
A better rephrasing:

"Instead of manually creating your objects via the DSL (e.g. the `Obj()` method), you may use the `Encoder` class to convert a user-defined type into the equivalent `Value type."